transformations/transformations.js

```diff
@@ -2,7 +2,7 @@ import { HOST_URL } from '../yasp.config';
 import moment from 'moment';
 
 // TODO move to utility module
-const isRadiant = (p) => p.player_slot < 128;
+const isRadiant = (playerSlot) => playerSlot < 128;
```

Comment:
In order to allow passing in player_match objects directly this function intentionally takes an object with `{player_slot: 128}`